pkgs/applications/misc/mediaelch/default.nix

```diff
@@ -38,6 +40,10 @@ mkDerivation rec {
     substituteInPlace MediaElch.pro --replace "/usr" "$out"
   '';
 
+  postInstall = ''
+    wrapProgram $out/bin/MediaElch --prefix LD_LIBRARY_PATH : "${lib.makeLibraryPath [ libmediainfo ]}"
```

Comment:
Please use patchelf.

Reply:
You can set `runtimeDependencies` to add libraries that are opened at runtime with `dlopen`.

Reply:
with `runtimeDependencies = [ (lib.getLib libmediainfo) ];` it does not work

Reply:
with `patchelf  --add-needed ${libmediainfo}/lib/libmediainfo.so.0  $out/bin/MediaElch` it does not compile: `patchelf: not an ELF executable`

Reply:
Is this a native executable or a script?

Reply:
its a c++ project, so it has to be compiled. But due to the QT-application it gets wrapped. the last line int the wrapper calls:
```
exec -a "$0" "/nix/store/5ksgxdrgyiaxj8ijcvfnzvbsidmi2kfw-mediaelch-2.8.14/bin/.MediaElch-wrapped"  "$@"
```

Reply:
Then it should at least use qtWrapperArgs or we double wrap which should be avoided https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/qt.section.md#locating-runtime-dependencies-qt-runtime-dependencies

Reply:
ok thanks for the suggestions. works now. better this way?

Reply:
The file is loaded here https://github.com/Komet/MediaElch/blob/v2.8.14/src/data/MediaInfoFile.cpp#L23 somehow. Could we patch that instead?

Reply:
not sure how to patch this. could you please provide an example?